zoomus/util.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, base_uri=None, timeout=15, **kwargs):
         self.timeout = timeout
         for k, v in kwargs.items():
             setattr(self, k, v)
+        self.session = requests.Session()
```

Comment:
This is a great start! For the sake of flexibility can we make this a configurable option and have the ability to start a new session. I.e. Let's make the method signature 
```
def __init__(self, base_uri=None, timeout=15, use_session=True, **kwargs):
```
Then we can do something like
```
self.session = requests.Session() ? use_session : requests
```
This should then either use the `Session` or `requests` depending on what was requested. 

We'd then also want to add a new method called something like `new_session` or something like that so that if someone needs a new session for some reason, they could do that without having to reinstantiate the `ApiClient`.

Reply:
Perhaps it might even be better if this let the client be used as a context manager, rather than having to explicitly say `use_session`. So then, reusing an example from the readme, a user could do something like:

```python
with ApiClient() as client:
    users = client.user.list().content
    meetings = {user["id"]: client.meeting.list(user_id=user["id"]).content for user in users}
```

This could be implemented by adding basically

```python
def __enter__(self):
    self.session = requests.Session()

def __exit__(self):
    self.session = None
```
And the individual `get`/`post`, etc, optionally call the client if it's present.

Reply:
Even better! Great idea!